Gemfile

```diff
@@ -186,6 +186,7 @@ group :test do
   gem 'rspec-repeat'
 
   gem 'simplecov', require: false
+  gem 'simplecov-lcov', require: false
```

Comment:
Change in Gemfile.lock is missing... Please run `bundle install`

Reply:
@matt-hh, see [095f35e](https://github.com/ComPlat/chemotion_ELN/pull/938/commits/095f35e3fbc5b941f9d4fe7f0bda3d8d0d6803cd).